### PR TITLE
handle more than one link and shorten long ones

### PR DESCRIPTION
**Pull request by pilz**
this follows up with #606 and makes it smarter.

### CODE REVIEW

**Comment by pilz** [comment]
jenkins please retest this

**Comment by pysailor** [comment]
jenkins please retest this

**Comment by pysailor** [comment]
Jenkins test this please

File: src/ploneintranet/core/browser/utils.py [hunk 1]
```diff
@@ -31,4 +31,11 @@ def link_urls(text):
...
     """
     urlfinder = re.compile('(http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+)')  # noqa
-    return urlfinder.sub(r'<a href="\1" target="_blank">\1</a>', text)
+    links = urlfinder.findall(text)
+    for link in links:
+        if len(link) > 40:
+            name = link[:40] + "&hellip;"
+        else:
+            name = link
+        text = text.replace(link, '<a href="%s">%s</a>' % (link, name))
```
Comment (pysailor) [inline src/ploneintranet/core/browser/utils.py:40]: @pilz Is it intentional that you remove `target="_blank"`?

**Pull request closed without merging**